tool/generator/lib/src/projection/utils.dart

```diff
@@ -160,14 +160,34 @@ String importForWin32Type(TypeIdentifier identifier) {
   }
 }
 
-/// Converts a namespace (e.g. `Windows.Win32.System.Console`) and returns the
-/// matching folder (e.g. `system/console`).
+/// Converts a namespace (e.g. `Windows.Win32.System.Console.CONSOLE_FONT_INFO`)
+/// and returns the matching folder (e.g. `system/console`).
 String folderFromNamespace(String namespace) {
```

Comment:
I think this is incorrectly named now? This is a fully-qualified type name, not a namespace. A namespace is a container. https://docs.microsoft.com/en-us/uwp/winrt-cref/winmd-files#namespace

Reply:
You're right. I was confused by the example in the comment "(e.g. `system/console`)" since this function returns `system` for `Windows.Win32.System.Console`. I guess the example should be "(e.g. `system`)".

Reply:
Great! Yeah, it's left over from some experimentation I did at projecting the entire Win32 API. I've more or less abandoned that for the time being, since so much of what else was generated was esoteric (fax services, HTML Help). I was experimenting with different levels of consolidation.